chroot-etc/croutonwheel.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
the original issue was that `croutonwheel` couldn't start until after `startunity` calls `exec gnome-session-wrapper ubuntu`, otherwise `xwininfo` complained and failed since X hasn't started yet.  use of `xwininfo` has since been replaced by `croutonwmtools`, and you mentioned that once the smartcycle branch is merged, `croutonwheel` should work when launched from `croutonxinitrc-wrapper`.  is this still the case?

trying to find a proper place to start `croutonwheel` reliably in unity was actually one of the more difficult parts of this PR. if this still won't work from `croutonxinitrc-wrapper`, the good news is that the `croutonwheel.desktop` solution (executed via `/etc/xdg/autostart`) is a solid alternative that plays well with ubuntu.